release/ray_release/config.py

```diff
@@ -15,6 +16,10 @@ class Test(dict):
     pass
 
 
+class TestDefinition(dict):
```

Comment:
maybe add a comment, like this is to store the parsed result of a yaml file somewhere, what fields / format are expected in general? 

`TestDefinition` feels like a really general name, and it has a simple but general implementation (dict), which I am not sure what exactly this is.